.changes/header.tpl.md

```diff
@@ -0,0 +1,6 @@
+# Changelog
```

Comment:
I think this file should be moved in some other central place so that they can be shared by multiple `.changie.yml` config files. Also the .changes should be one per project since we want to use changie for all the projects in the monorepo. There are just way to easy to make mistakes in changelogs today and getting the conflicts between PRs. They are far less in libraries since we don't change them that often still having a tool would help us having the right format.